Binary search.java

```diff
@@ -0,0 +1,38 @@
+import java.util.*;  
+ class Binarysearch {  
+public static void main(String[] args) {  
```

Comment:
Please do add a space after the import and provide proper indentation. After you are done with these changes, I will approve this PR.

Reply:
okay